website/docs/cloud-docs/integrations/service-now/index.mdx

```diff
@@ -76,12 +76,9 @@ To start using Terraform with the ServiceNow Catalog Integration, you must have:
 
 This integration has been tested on the following ServiceNow server versions:
```

Comment:
```suggestion
You can use this integration on the following ServiceNow server versions:
```

Suggest changing "This integration has been tested" to "You can use this integration" to have an active voice.

Reply:
Good suggestion, thanks! Done.